tests/tests_lite/plugins/precision/test_double.py

```diff
@@ -0,0 +1,25 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import torch
+
+from lightning_lite.plugins.precision.double import DoublePrecisionPlugin
+
+
+def test_double_precision_forward_context():
```

Comment:
(meta question, can resolve): How are you tackling tests?

Are these moved from PL? If yes, will you remove them later?
If no: will they be duplicated?
I wonder what will be the coverage overlap between PL and Lite once this is finished.

Reply:
> Are these moved from PL? If yes, will you remove them later?

Some of them are, some of them are newly written by me. 


My workflow is:

1. Copy the original test file to the lite test folder. 
2. Delete all tests that require Trainer or similar PL interfaces, keep all unit tests that just test the component
3. Write new  unit tests that were not there in PL, to cover the important features and APIs of the component

After Lite has completely moved and all components have been integrated, a follow up workflow will be:

1. Rewrite integration tests that use the Trainer to Lite
2. Delete tests in PL that cover functionality completely implemented in Lite.

Over the next few weeks, we will see a very strong increase in tests in Lite with a small decrease in tests on PL. During the transition, lots of tests will be duplicated. When we remove redundant tests from PL, we want to make sure coverage does not decrease.

Reply:
> Delete all tests that require Trainer or similar PL interfaces, keep all unit tests that just test the component

Since we often use the "train-model-to-test" pattern by putting assertions inside a hook, this most tests will get deleted, making 

> Rewrite integration tests that use the Trainer to Lite

a big task down the line.

Thanks for the description!